.github/workflows/windows_workflow.yml

```
@@ -0,0 +1,35 @@
+name: windows-workflow
+
+on:
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.5.0
+      with:
+        access_token: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/checkout@v2
+    - uses: gradle/wrapper-validation-action@v1
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 15
+    - uses: actions/cache@v2
+      with:
+        path: ~/.gradle/caches
+        key: ${{ runner.os }}-2-gradle-${{ hashFiles('**/*.gradle*') }}
+        restore-keys: |
+          ${{ runner.os }}-2-gradle-
+
+    - name: Gradle clean build
+      uses: eskatos/gradle-command-action@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        HEAD_REF: ${{ github.head_ref }}
```

I think you need to add this to get it to be succesful:
```suggestion
        HEAD_REF: ${{ github.head_ref }}
        FLANK_PROJECT_ID: flank-open-source
```
or possibily 
```suggestion
        HEAD_REF: ${{ github.head_ref }}
        GOOGLE_CLOUD_PROJECT: flank-open-source
```

---

this task run only local unit tests, so I do not think we need to specify project id

---

@piotradamczyk5 the run is failing as the project field is null. I assume the project field requires this correct?
https://github.com/Flank/flank/pull/1534/checks?check_run_id=1767430233

---

@piotradamczyk5 I see a lot of errors actually nevermind.

---

I will check them, they could be related to other things